inst/shiny-examples/custom_css/app.R

```diff
@@ -0,0 +1,47 @@
+## ---- custom-css-app ------------------------------------------------
+## Example shiny app with custom css
+
+library(shiny)
+library(sortable)
+
+ui <- fluidPage(
+  fluidRow(
+    column(
+      width = 12,
+      tags$b("Exercise"),
+      rank_list(
+        text = "Drag the items in any desired order",
+        labels = list(
+          "one",
+          "two",
+          "three",
+          htmltools::tags$div(
+            htmltools::em("Complex"), " html tag without a name"
+          ),
+          "five" = htmltools::tags$div(
+            htmltools::em("Complex"), " html tag with name: 'five'"
+          )
+        ),
+        input_id = "rank_list_1",
+        class = c("default-sortable custom-sortable") # add custom style
```

Comment:
Can this be done as `c("default-sortable", "custom-sortable")`? We auto collapse the class values.

Reply:
Nice.  Will do.